app/src/styles.sass

```diff
@@ -164,17 +163,52 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
   right: 0
   display: flex
   flex-flow: row nowrap
-  width: 80%
+  overflow: hidden
+  width: 80vw
+  height: 150vw
   @media screen and (min-width: $breakpoint-tablet) and (max-width: $breakpoint-laptop)
-    width: 50%
+    width: 50vw
+    height: 70vw
   @media screen and (min-width: $breakpoint-laptop)
     display: none
-  .info-panel-icon #help-button
-    margin: 0.7em $small-space
+  .info-panel-content
+    position: absolute
+    z-index: 1
+    right: 0
+    width: 70vw
```

Comment:
I think that setting this width here works only because `position: absolute` removes the element from the flex container. Inside a flex, if the flex has a fixed width, the width of the element it contains cannot be fixed. Since using `position: absolute` on .info-panel-content results in many issues, I suggest you to remove it, and to fix the width of something *inside* .info-panel-content: either .information (you'll have to remove `display: contents`) or a new div still to add.